### DISCUSSION TITLE
XCode build unable to auto detect certificate and provisioning profile

### DISCUSSION THREAD

**Discussion by krunaldarji** [codemagic-ci-cd/codemagic-docs#2171] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

1.Execute iOS Build with REST API
2.Download all required certificates and profiles through command
3.Hit "xcode-project use-profiles" command but unable to select available profiles

### Expected results

Should able to autoselect provisioning profiles of "OnlinekakaCustomer" and "ImageNotification" extension targets.

### Actual results

Fail to autoselect profiles

### Build id (optional)

6545fe80931659eb1762fa54

**Comment by icarusdust** [comment]
Hi @krunaldarji, we are discussing the same issue in Intercom with Bilal from your team.

**Comment by icarusdust** [comment]
It has already been fixed! 

When entry was included during code signing, default `PROVISIONING_PROFILE_SPECIFIER` from `.pbxproj `was not removed.
